### PR TITLE
Tasks tab on assign to me and go to tasks fix

### PR DESCRIPTION
**Pull request by connorpgpmcelroy**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-6757

### Change description ###
Fixing issue whereby 'assign to me and go to case' does not go to tasks tab properly on the case. Caused by route mistake

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x ] No
```

### CODE REVIEW

**Review by DavidJayakumar — APPROVED** [review]
Look good

**Review by Surekha-HMCTS — APPROVED** [review]
Looks good

File: config/default.json [hunk 1]
```diff
@@ -82,7 +82,7 @@
     "s2s": "http://rpe-service-auth-provider-prod.service.core-compute-prod.internal",
     "termsAndConditions": "http://xui-terms-and-conditions-aat.service.core-compute-aat.internal",
     "payment_return_url": "https://paymentoutcome-web.prod.platform.hmcts.net/",
```
Comment (udaydenduluri33) [inline config/default.json:84]: Are you sure about this change? If you doing this then you need to change Reference.ts accordingly

Comment (connorpgpmcelroy) [reply config/default.json:84]: Yes, this is what is actually within Reference - unfortunately due to people's merge conflicts I think this file keeps changing back - it should be judicialBookingApi in all instances

**Review by udaydenduluri33 — APPROVED** [review]

**Pull request merged**